assets/components/circlesIntroduction/circlesIntroduction.jsx

```diff
@@ -18,39 +25,23 @@ type PropTypes = {
 function CirclesIntroduction(props: PropTypes) {
   return (
     <section className="component-circles-introduction">
+      <SvgCirclesHeroDesktop />
+      <SvgCirclesHeroMobileLandscape />
+      <SvgCirclesHeroMobile />
       <div className="component-circles-introduction__content gu-content-margin">
-        <h1 className="component-circles-introduction__heading">
-          {props.headings.map(heading =>
-            <span className="component-circles-introduction__heading-line">{heading}</span>)}
-        </h1>
-        <Highlights highlights={props.highlights} />
+        <div className="component-circles-introduction__copy">
```

Comment:
The class name including `copy` in `component-circles-introduction__copy` feels weird since inside of it there is no copy but a title. Additionally, I believe "copy" is a word from the editorial world. I would rename it with `component-circles-introduction_content`. What do you think?

Reply:
`component-circles-introduction__content` is the parent element.

What's the issue with copy? We've used it [throughout the codebase](https://github.com/guardian/support-frontend/search?utf8=✓&q=copy&type=) to refer to editorial copy.